lib/model/picture.dart

```diff
@@ -1,20 +1,13 @@
-class Picture{
-
-  late int? id;
-  late String? pictureName;
-
-  Picture(int i, String imgString, { required this.id, required this.pictureName});
-
-  Map<String, dynamic> toMap(){
-    var map = <String, dynamic> {
-      'id' : id,
-      'pictureName' : pictureName,
-    };
-    return map;
-  }
-
-  Picture.fromMap(Map<String, dynamic>map) {
-    id = map['id'];
-    pictureName = map['pictureName'];
-  }
-}
\ No newline at end of file
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:flutter/foundation.dart';
+
+part 'picture.freezed.dart';
```

Comment:
remove the declaration which was not generated. There could not be .freezed and .g files at the same  time

Reply:
fixed successfully